packages/dashboard/src/components/resourceExplorer/nextResourceExplorer.tsx

```diff
@@ -0,0 +1,179 @@
+    isHeader: true,
+  };
+
+  const navigateToBranch = (item: EitherAssetSummary) => {
+    item.hierarchies?.forEach((hierarchy: AssetHierarchy) => {
+      provider?.expand(new BranchReference(item.id, hierarchy.id as string));
+    });
+    if (item?.id) {
+      setCurrentBranchId(item.id);
+    }
+  };
+
+  const handleCrumbClick = (item: AssetSummary) => {
+    navigateToBranch(item);
+  };
+
+  const handlePanelItemClick = (item: ExtendedPanelAssetSummary) => {
+    navigateToBranch(item as EitherAssetSummary);
+    const nextCrumbs = crumbs;
+    nextCrumbs.push(item as EitherAssetSummary);
+    setCrumbs(nextCrumbs);
+  };
+
+  const setCrumbsToSearch = () => {
+    setCrumbs([
+      {
+        name: messageOverrides.resourceExplorer.searchQueryHeader,
+        isHeader: true,
+      } as EitherAssetSummary,
+    ]);
+  };
+
+  const getCachedCurrentAssetProperties = async (currentBranchId: string) => {
+    if (currentBranchId && hasKey(currentBranchId)) {
+      return cache[currentBranchId];
+    }
+    const currentAssetProperties = await getCurrentAssetProperties(currentBranchId, messageOverrides, client);
+    update(currentBranchId, currentAssetProperties);
+    return currentAssetProperties;
+  };
+
+  useEffect(() => {
+    (async () => {
+      const [currentAssets, currentAssetProperties, describedAsset] = await Promise.all([
+        getCurrentAssets(provider, currentBranchId, messageOverrides),
+        getCachedCurrentAssetProperties(currentBranchId),
+        describeCurrentAsset(currentBranchId, client),
+      ]);
+
+      const nextAlarms = retrieveAlarms(describedAsset);
+      setAlarms(nextAlarms);
+
+      const nextPanelItems = currentAssetProperties.concat(currentAssets);
+      const nextPanelItemsWithAlarmsHeader = [...nextPanelItems, ...(nextAlarms.length > 0 ? [alarmsHeaderItem] : [])];
+      setPanelItems(nextPanelItemsWithAlarmsHeader);
+    })();
+  }, [JSON.stringify(provider?.branches), JSON.stringify(provider?.assetNodes), currentBranchId]);
+
+  const ResourceExplorerAssetHierarchy = () => (
+    <Box padding='l'>
+      <SpaceBetween size='s'>
+        <ResourceExplorerSearchbar
+          provider={provider}
+          assetPropertiesCache={cache}
+          setCrumbsToSearch={setCrumbsToSearch}
+          setPanelItems={setPanelItems}
+          messageOverrides={messageOverrides}
+        />
+
+        <ResourceExplorerBreadcrumbs crumbs={crumbs} setCrumbs={setCrumbs} handleCrumbClick={handleCrumbClick} />
+
+        <ResourceExplorerPanel
+          alarms={alarms}
+          panelItems={panelItems}
+          handlePanelItemClick={handlePanelItemClick}
+          messageOverrides={messageOverrides}
+        />
+      </SpaceBetween>
+    </Box>
+  );
+
+  const ResourceExplorerPropertySearch = () => (
+    <Box padding='l'>
+      <ResourceExplorerPropertySearchbar messageOverrides={messageOverrides} />
+    </Box>
+  );
+
+  return (
+    <div className='iot-resource-explorer'>
```

Comment:
is this component used? I thought we removed the tabs?

Reply:
No, removed.